src/components/Main.vue

```diff
@@ -75,6 +75,26 @@
                                 label="Second step" />
                         </div>
                     </div>
+                    <div class="row">
+                        <div class="col">
+                        <InputRange
+                            v-model="paramX"
+                            v-bind:min="-0.99"
+                            v-bind:max="0.99"
+                            setp="0.01"
```

Comment:
The prop name is `step`.

Reply:
Also, please bind the float value so it's correctly passed to the component (see `min`/`max` above).